pkg/utils/cluster.go

```diff
@@ -51,3 +51,22 @@ func openshiftSingleNodeClusterStatus(c client.Client) (bool, error) {
 	}
 	return infra.Status.ControlPlaneTopology == configv1.SingleReplicaTopologyMode, nil
 }
+
+func IsHypershiftCluster(c client.Client) (bool, error) {
+	if os.Getenv("CLUSTER_TYPE") == ClusterTypeOpenshift {
+		return openshiftExternalControlPlaneStatus(c)
+	}
+	return false, nil
+}
+
+func openshiftExternalControlPlaneStatus(c client.Client) (bool, error) {
```

Comment:
I'm wondering if  `openshift` is needed in the name

Reply:
I can remove openshift!

Reply:
although actually this models the other functions in the directory like `openshiftSingleNodeClusterStatus`. So happy to create a new path but it would be different from previous declarations (and this is a specific concept to Openshift.)

Reply:
Ok, thanks for that. Uniform usage is good too. It's a small nit.

In any case, github.com/k8snetworkplumbingwg/sriov-network-operator/pull/ also has something very similar called `IsExternalControlPlaneCluster`. Can you please create something that both PRs can use?

Reply:
yes! I will bring those both together!